src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Lottery {
+    public Colors getRandomColor() {
+        return Colors.values()[new Random().nextInt(Colors.values().length)];
```

Comment:
create a global Random field and use it here, so you won't need to create its instance every time when you call this method